share/spice/snow/snow.js

```diff
@@ -45,11 +45,20 @@ function nrio (api_result) {
                 content: Spice.snow.content,
                 moreAt: true
             }
+        },
+        onShow: function(){
+         // add the snowflakes, if it's snowing
+         if(!api_result.is_snowing){
+             return; 
+             }
```

Comment:
Indentation looks off here. Please make sure you're indented with soft-tabs (4 space)